source/open-definition-revision-process-dev.markdown

```diff
@@ -2,19 +2,21 @@
 Definition Revision Approval Process _- draft_
 ====================================
 
-1. Issues with the current Open Definition are identified.  The Open Definition community will disucss on the mailing list and decide which issues are to be dealt with in the next release reach consensus on each of these issues.
+1. Issues with the current Open Definition are identified.  The Open Definition community will disucss on the mailing list or discuss.okfn.org forum and decide which issues are to be dealt with in the next release.
 
 2. A new draft document is started.  Chair notifies the list and other relevant fora that a new draft document has been started summarizing the issues that the upcoming release is to address and inviting participation.
 
-3. Issues with the current Open Definition are identified.  The Open Definition community will discuss on the mailing list and decide which issues are to be dealt with in the next release reach consensus on each of these issues.  The draft document is updated to relfect current consensus.
+3. Along with discussion, people submit pull requests to the draft until it seems to relfect current consensus.
 
-4. The Open Definition Advisory Council chair will summarize the consensus to the Advisory Council on the mailing list and to other relevant fora, okfn-discuss at a minimum.
+4. When no outstanding issues remain unsettled, the Open Definition Advisory Council chair will summarize the consensus to the Advisory Council on the mailing list and to other relevant fora, okfn-discuss at a minimum.
```

Comment:
I don't see how "When no outstanding issues remain unsettled" is practical.  There are always going to be outstanding issues.  I think someone needs to draw the line somewhere and say "this is in" and "everything else is for later", typically related to the intention of the release.  I think it's up to the chair to draw that line and then it is up to others to protest before a vote is called if they want to object.